schematools/cli.py

```diff
@@ -27,7 +27,7 @@
     revoke_permissions,
 )
 from .provenance.create import ProvenaceIteration
-from .types import DatasetSchema, SchemaType
```

Comment:
Ik twijfel of module moet echt hernoemt worden.
 - Ja, naam collisie met builtin module is niet mooi
 - `from schematools.types` oplosst probleem met imports
 
Zo een simpele verandering kan 2 andere applicaties breken:
 - https://github.com/Amsterdam/dso-api/search?q=from+schematools.types
 - https://github.com/Amsterdam/dataservices-airflow/search?q=from+schematools+types

Reply:
Eens met Nick. Omdat schematools een library is zou ik voorzichtig willen zijn met introductie van breaking changes. Overigens zou een relatieve import als `.types` door Pycharm toch niet als een absolute import geinterpreteerd mogen worden?

Reply:
vreemd, want juist in Python 3 is het probleem met relative/absolute imports opgelost. Staat in PyCharm jouw project root 1 map te diep? Ik heb dit probleem niet.

Reply:
Ik ga m'n PyCharm instellingen er  nog even op na lopen. Mocht dan niets opleveren, maak ik er een absolute import van.

Reply:
@vdboor Yep, stond één map te diep. Change reverted.

@ngaranko Goed punt dat het andere applicaties had gebroken. Zal daar op letten in het vervolg.